gnovm/pkg/gnolang/ownership.go

```diff
@@ -112,15 +114,16 @@ type Object interface {
 	SetIsDeleted(bool, uint64)
 	GetIsNewReal() bool
 	SetIsNewReal(bool)
+	GetBoundRealm() PkgID
```

Comment:
The word "bound" is not used in the inter-realm spec; it is "attached" to the realm.
Confusing vs BoundMethod